src/queries/constructionSite.js

```diff
@@ -0,0 +1,50 @@
+import gql from 'graphql-tag';
+
+export const GET_CONSTRUCTION_SITES = gql`
+  query GenericItems($ids: [ID]) {
+    constructionSites: genericItems(ids: $ids, genericType: "ConstructionSite") {
+      id
+      title
+      contentBlocks {
+        id
+        body
+      }
+      categories {
+        id
+        name
+      }
+      dataProvider {
+        id
+        name
+        dataType
+      }
+      locations {
+        id
+        name
+        geoLocation {
+          latitude
+          longitude
+        }
+      }
+      mediaContents {
+        id
+        captionText
+        contentType
+        copyright
+        sourceUrl {
+          id
+          url
+        }
+      }
+      dates {
+        id
+        dateEnd
+        dateStart
+        timeDescription
+        timeEnd
+        timeStart
```

Comment:
do we need that time values?
are there other values, which we do not use on the construction screen?

Reply:
i don't think we make use of them at the moment, so i removed them for now.

the copyright and captiontext of the mediacontent were also not being used, but instead of removing them i added them to the image component.